docs/source/notebooks/mmm/dag

```diff
@@ -0,0 +1,10 @@
+digraph {
+	graph [dpi=200]
+	media [label="media spend"]
+	seas [label=seasonality]
+	trend [label=trend]
+	y [label=sales]
+	media -> y
+	seas -> y
+	trend -> y
+}
```

Comment:
Why this change?

Reply:
It wasn't intentional - I was updating my dev dependencies via the Makefile and accidentally ran the UML hook and notebook runner. Want me to remove these files?

Reply:
No, it think it should be fine :)